executor/builder.go

```diff
@@ -2138,12 +2138,42 @@ func (b *executorBuilder) buildIndexLookUpJoin(v *plannercore.PhysicalIndexJoin)
 	for i := 0; i < len(v.OuterJoinKeys); i++ {
 		outerKeyCols[i] = v.OuterJoinKeys[i].Index
 	}
-	e.outerCtx.keyCols = outerKeyCols
 	innerKeyCols := make([]int, len(v.InnerJoinKeys))
 	for i := 0; i < len(v.InnerJoinKeys); i++ {
 		innerKeyCols[i] = v.InnerJoinKeys[i].Index
 	}
+	e.outerCtx.keyCols = outerKeyCols
 	e.innerCtx.keyCols = innerKeyCols
+	outerHashCols, innerHashCols := make([]int, len(outerKeyCols)), make([]int, len(innerKeyCols))
+	copy(outerHashCols, outerKeyCols)
+	copy(innerHashCols, innerKeyCols)
+
+	// we can use the `col <eq> col` in `OtherCondition` to build the hashtable to avoid the unnecessary calculating.
```

Comment:
It's better to show eq condition in operator info of explaining result like Hash Join.